lebedigital/openbis/README.md

```diff
@@ -0,0 +1,140 @@
+
+## Glossary
+
+- **Interbis** - A Subclass of the Openbis class in the pybis library. It is used for connecting to the openBIS Database and for all Database queries like getting an overview of the Database
+- **ExpStep** - A Class for working with openBIS Experimental Steps. Used for defining Objects which describe real Experiments that you want to describe with metadata and upload to the openBIS Database
+- **Space** - The Highest "directory" abstraction in the Datastore. Contains all other openBIS "directories". One Space is defined per User or per Group.
+  - _Example_: Some Paper about compressing concrete
+- **Project** - Second highest "directory" abstraction in the Datastore. Contained in a Space, you can define infinitely many of them.
+  - _Example_: Concrete_Compression
+- **Collection / Experiment** - Third highest "directory" abstraction in the Datastore. Contained in a Project, you can define infinitely many of them.
+  - _Example_: Concrete_Compression_Tests
+- **Sample / Object** - An abstract representation of your experiment with your defined metadata **also called Experimental Steps** in the openBIS Database. Contained in a Collection, you can define infinitely many of them.
+  - _Example_: Concrete_Compression_Experiment_17_10_2022
+- **Dataset** - An openBIS abstraction containing your data files like Excel sheets or csvs.
+  - _Example_: Concrete_Compression_Experiment_17_10_2022_RAW_DATA
+- **Sample / Object Type** - A blueprint you need to define in order to upload Samples/Objects of real world experiments.
+  - _Example_: EXPERIMENTAL_STEP_CONCRETE
+- **Property** - One piece of metadata you use do describe your Samples/Objects. 
+  - _Example_: Name of your sample
+- **Parents** of a Sample - You can define another Sample to be a parent of a different sample.
+  - _Example_: A concrete mixture could be a **parent** of a concrete compression experiment
+
+> You can look at the definition from pybis [here](https://pypi.org/project/PyBIS/)
+
+## Structure
+
+### Interbis
+
+Is an extension of the existing Openbis class defined in pybis and as such extends all pybis functionality.
+Interbis provides additional functionality for getting an overview of the datastore and find out what the structure of the Database looks like.
+Interbis also makes it easier to search for existing samples and retrieve their Identifiers from their metadata like Name.
+
+### ExpStep
+
+Is a Class which provides functionality to define Samples as they are accepted by pybis and in turn the openBIS datastore.
+Example uses of the Class include uploading and retrieving samples and datasets from the Database.
+I recommend deriving the class in your personal workflow and tweaking it to make it easier to write data processing and uploading scripts. 
+
+
+## Example Workflow of Interbis/ExpStep - Concrete Measuring Project
+
+### Start with connecting yourself to the Database
```

Comment:
I wouldn't right it in the you-form.
Maybe numbering the steps and then write 
1. Connecting to Database
2. Create new object/sample type "Experimental Step XY"
3. Create and upload objects/samples from type "Experimental Step XY"
4. Upload corresponding datasets